src/main/java/com/gluonhq/substrate/util/windows/MSIBundler.java

```diff
@@ -0,0 +1,325 @@
+            FileOps.deleteDirectory(tmpMSI);
+        }
+        Files.createDirectories(tmpMSI);
+
+        Path userAssets = rootPath.resolve(Constants.WIN_ASSETS_FOLDER);
+        // copy assets to gensrc/windows
+        Path windowsGenSrcPath = paths.getGenPath().resolve(sourceOS);
+        Path windowsAssetPath = windowsGenSrcPath.resolve(Constants.WIN_ASSETS_FOLDER);
+        Files.createDirectories(windowsAssetPath);
+        // Copy system resources and over-write it with user provided resources
+        FileOps.copyDirectoryFromResources("/native/windows/assets", windowsAssetPath);
+        Logger.logInfo("Default icon.ico image generated in " + windowsAssetPath + ".\n" +
+                "Consider copying it to " + rootPath + " before performing any modification");
+        if (Files.exists(userAssets)) {
+            FileOps.copyDirectory(userAssets, windowsAssetPath);
+        }
+
+        Path config = tmpMSI.resolve("config");
+        Files.createDirectories(config);
+        Path wixPath = config.resolve("main.wxs");
+        Path wixObjPath = wixPath.getParent().resolve(wixPath.getFileName() + ".wixobj");
+        Path msiPath = paths.getGvmPath().getParent().resolve(appName + "-" +  version  + ".msi");
+        FileOps.copyResource("/native/windows/wix/main.wxs", wixPath);
+        FileOps.copyDirectory(windowsAssetPath, config);
+
+        /**
+         * Wix Compile
+         */
+        Map<String, String> userInput = createAppDetailMap();
+        List<String> processArgs = new ArrayList<>(List.of(
+                WixTool.CANDLE.getPath(),
+                "-nologo",
+                wixPath.toString(),
+                "-ext", "WixUtilExtension",
+                "-arch", "x64",
+                "-out", wixObjPath.toString()
+        ));
+
+        userInput.entrySet().stream()
+                .map(wixVar -> String.format("-d%s=%s", wixVar.getKey(), wixVar.getValue()))
+                .forEachOrdered(processArgs::add);
+        Logger.logInfo(String.join(" ", processArgs));
+        ProcessRunner candle = new ProcessRunner(processArgs.toArray(new String[0]));
+        if (candle.runProcess("Wix Compiler") != 0) {
+            throw new IOException("Error running candle to generate wixobj");
+        }
+
+        /**
+         * Wix Link
+         */
+        processArgs.clear();
+        processArgs.addAll(List.of(
+                WixTool.LIGHT.getPath(),
+                "-nologo",
+                "-spdb",
+                "-sw1076", // https://github.com/wixtoolset/issues/issues/5938
+                "-ext", "WixUtilExtension",
+                "-ext", "WixUIExtension",
+                "-out", msiPath.toString(),
+                wixObjPath.toString()
+        ));
+
+        ProcessRunner light = new ProcessRunner(processArgs.toArray(new String[0]));
+        if (light.runProcess("Wix Linker") != 0) {
+            throw new IOException("Error running light to generate msi");
+        }
+
```

Comment:
Add a final message like in https://github.com/gluonhq/substrate/blob/master/src/main/java/com/gluonhq/substrate/util/macos/Packager.java#L243